test/e2e/chaos/pod.go

```diff
@@ -214,3 +328,12 @@ func waitDeploymentReady(name, namespace string, cli kubernetes.Interface) error
 		return true, nil
 	})
 }
+
+func getUIDs3Pods(pods []corev1.Pod) [3]string {
```

Comment:
why we limit 3 here. I think this is not necessary.

Reply:
I was trying to return an array here but it looks like there's no way to return a not-fixed-length array in a go function. Is there some better design for this kind of scenario?

Reply:
I think you could build a function which receives 2 pod array and compare whether they have same uid and return the boolean result.

Reply:
I see